docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -1,11 +1,456 @@
 # Hello Triangle
 
-## Graphics Pipeline
+In the last chapter we initialized core components of D3D11 and DXGI such as the Device and the SwapChain, but simply clearing the Window with some color is pretty boring.
+This time we'll be drawing our first triangle with a nice froge-like color.
 
-!!! error "explain in an overview fashion with pics what directx pipeline is"
+## The Pipeline
+The fundamental part of all graphic APIs is the "Graphics Pipeline", whether we're drawing a single triangle, a textured frog or the whole Elden Ring map, everything goes through the pipeline; it is a series of functions that are either hardwired in hardware, configurable or fully programmable, and all it does is transform all that we draw from 3D space to the 2D space that is our monitor. All the steps in the graphics pipeline go from top to bottom and are shown below.
 
-    how it roughly works and what it means and can do
+![](https://docs.microsoft.com/en-us/windows/win32/direct3d11/images/d3d11-pipeline-stages.jpg)
```

Comment:
Is it somehow possible to have this diagram remade so that it
- looks a bit better and
- supports transparent background somehow (while still readable in light/dark mode)???

We should consult with the peeps in the channel maybe they have an idea how to do that (or media can be toggled depending on light/dark mode and we have one with white lines for dark and black for light mode etc